website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -24,36 +27,66 @@ export default function ConsulOnKubernetesPage() {
         }}
       />
 
-      {/* side by side section */}
-      {/* block list will be a node within the sidebyside section once that is complete */}
       <section>
-        <BlockList
-          blocks={[
-            {
-              title: 'Multi-Cluster environments',
-              description:
-                'Organizations typically prefer to utilize a more distributed model for Kubernetes deployments. Rather than maintain a single cluster, they connect multiple environments for testing, staging, and production purposes.',
-              image: require('./images/blocks/multi-cluster.svg'),
-            },
-            {
-              title: 'Connecting K8s to non-K8s',
-              description:
-                'Creating consistency when connecting Kubernetes to non-Kubernetes environments can be challenging, workflows need additional automation to accommodate many virtual machines or containers.',
-              image: require('./images/blocks/connecting.svg'),
-            },
-            {
-              title: 'Securing K8s networking',
-              description:
-                'Securing Kubernetes networking with multiple layers of network policies can be challenging. Policies can be handled at the application layer, container/OS or at the networking level. ',
-              image: require('./images/blocks/securing.svg'),
-            },
-            {
-              title: 'Kubernetes Monitoring',
-              description:
-                'Obtaining insights on what’s going on and the health of Kubernetes clusters can be complicated. In addition, security issues and vulnerabilities need to be properly tracked.  ',
-              image: require('./images/blocks/monitoring.svg'),
-            },
-          ]}
+        <SideBySide
+          left={
+            <div>
```

Comment:
If this div doesn't do anything other than just allow you to pass in multiple elements here, you could use a fragment.  `<></>`, to accomplish the same thing.

Cool with the extra div though if it serves ya a purpose.